components/molecules/charts/schedule/helper/prepareHiddenColumn.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Components - Molecules - Charts - Schedule - Helper - Hidden Column
+ */
+
+export default (hiddenColumn) => {
```

Comment:
@Ojaghi1991 I think proptypes can still work on non react methods - I have used them, prob need to test but lets see